### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 3 - Esercizio 2

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#500] [Esercizi M-Z]
![es.2](https://github.com/user-attachments/assets/aefa76ce-d85a-486e-9ef2-7a8216f08745)

**Comment by AglaiaNorza** [comment]
potrebbe essere sensato
![es.2](https://github.com/user-attachments/assets/12382421-7d1f-40c2-928c-4f205b042eb6)

**Comment by erinrrr** [comment]
![image](https://github.com/user-attachments/assets/d0cd5758-7ce1-45bf-afd1-fec8f6d4e5c5)

**Comment by marcoseverini** [comment]
<img width="670" alt="immagine" src="https://github.com/user-attachments/assets/0bb1ca5a-3893-47a8-91f0-9f414175f3c1">
<img width="800" alt="immagine" src="https://github.com/user-attachments/assets/3a724fd5-7ccd-4018-aa62-842b9dd51825">
<img width="586" alt="immagine" src="https://github.com/user-attachments/assets/88c85776-37c0-4223-b704-05844514bbb9">

**Comment by alem1105** [comment]
![image](https://github.com/user-attachments/assets/1858713f-9998-4e60-82f8-13752667f727)
